src/book.py

```diff
@@ -435,7 +447,7 @@ def _read_coinbase_v2(self, file_path: Path) -> None:
         self._read_coinbase(file_path=file_path)
 
     def _read_coinbase_pro(self, file_path: Path) -> None:
-        platform = "coinbase_pro"
+        platform = "coinbase"
```

Comment:
As discussed in #138, instead of making them the same, we should consider the deposits/withdrawals between them.